doc/openocd.texi

```diff
@@ -10871,6 +10871,18 @@ Control dcsr.ebreaku. When on (default), U-mode ebreak instructions trap to
 OpenOCD. When off, they generate a breakpoint exception handled internally.
 @end deffn
 
+@deffn {Command} {riscv eq_match_trigger_enable} [on|off]
+When on, allow OpenOCD to use equality match trigger in wp.
+@end deffn
+
+@deffn {Command} {riscv napot_trigger_enable} [on|off]
+When on, allow OpenOCD to use NAPOT trigger in wp.
+@end deffn
+
+@deffn {Command} {riscv ge_lt_trigger_enable} [on|off]
+When on, allow OpenOCD to use GE/LT triggers in wp.
+@end deffn
+
```

Comment:
I have tried to make the text more clear to the reader. Let me know what you think.

```suggestion
The commands below can be used to prevent OpenOCD from using certain RISC-V trigger features, for example in cases when there are known issues in the target hardware.

@deffn {Command} {riscv eq_match_trigger_enable} [on|off]
When on (default), allow OpenOCD to use exact-match triggers in watchpoints.
@end deffn

@deffn {Command} {riscv napot_trigger_enable} [on|off]
When on (default), allow OpenOCD to use NAPOT trigger in watchpoints.
@end deffn

@deffn {Command} {riscv ge_lt_trigger_enable} [on|off]
When on (default), allow OpenOCD to use a pair of chained less-than & greater-than triggers in watchpoints.
@end deffn

```

Reply:
Addressed